dockerfiles/custom_images.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
```

Comment:
in future that script will check if targets are there and other things which is needed only for devs so I don;t know if we should have everything in build.include
for me build.include is source for build.sh but I using build.sh to do some other things.